src/main.rs

```diff
@@ -233,6 +234,55 @@ fn color_for_level(lvl: &JobLogLevel) -> Color {
     }
 }
 
+enum JobBehaviour {
+    CancelLava,
+    LeaveRunning,
+}
+
+struct JobCanceller {
+    behaviour: JobBehaviour,
+    lava: Arc<ThrottledLava>,
+}
```

Comment:
Not sure if this should have a seperate object; it seems to add little value as opposed to just have a cancel_behaviour field on the Run struct